app/controllers/course/assessment/submission/answer/comments_controller.rb

```diff
@@ -7,7 +7,23 @@ class Course::Assessment::Submission::Answer::CommentsController < \
 
   delegate :discussion_topic, to: :@answer
 
+  def create
+    @answer.class.transaction do
+      @post.title = @assessment.title
+      if super && @answer.save
```

Comment:
This is called in `Course::Discussion::PostsConcern#create`. When a user comments on an answer, he will be automatically subscribed to it.